40th.html

```diff
@@ -54,7 +54,24 @@ <h3 class="panel-title">Event Calendar</h3>
 			  </div>
 		  </div>
 		</div>
-		<div class="panel panel-default">
+		<div class="panel panel-csh">
+		  <div class="panel-heading">
+		    <h3 class="panel-title">Hotel Reservations</h3>
+		  </div>
+		  <div class="panel-body">
+		  	<div class="row">
+		  		<div class="col-md-2">
+		  			<img src="resources/images/40th/hampton.png" alt="Hampton by Hilton" width="100%">
+		  		</div>
+		  		<div class="col-md-10">
+		  			<p>CSH has reserved a block of rooms at the Hampton Inn & Suites Rochester/Henrietta for the event. The hotel has created a reservation portal for members to reserve rooms in our hotel block at a special rate. Rooms start at $99/night until March 15th or until the block is sold-out, whichever comes first.</p>
```

Comment:
Fixed in https://github.com/stevenmirabito/CSHPublicSite/commit/dd058014bb224d92f55032c314bdec46cb46245a